packages/components/pie-checkbox/src/index.ts

```diff
@@ -84,20 +125,21 @@ export class PieCheckbox extends RtlMixin(LitElement) implements CheckboxProps {
         } = this;
 
         return html`
-        <label>
+        <label data-test-id="checkbox-component">
             <input
                 type="checkbox"
-                ?checked=${ifDefined(checked)}
-                .value=${ifDefined(value)}
+                .value=${value}
+                ?checked=${checked}
```

Comment:
Does this need to use [live](https://lit.dev/docs/templates/directives/#live)? I'm not sure but worth checking if you haven't already